src/components/modalToolbar/AposModalToolbar.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div :class="className" class="apos-toolbar">
+    <div v-if="hasLeftControls" class="apos-toolbar__group apos-toolbar__group--left">
+      <slot name="leftControls"></slot>
+    </div>
+    <div v-if="hasRightControls" class="apos-toolbar__group apos-toolbar__group--right">
+      <slot name="rightControls"></slot>
+    </div>
+  </div>
+</template>
+
+<script>
+
+export default {
+  components: { 
+  },
+  props: {
+    className: String
+  },
+  computed: {
+    hasLeftControls() {
+      return !!this.$slots.leftControls;
+    },
+    hasRightControls() {
+      return !!this.$slots.rightControls;
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+  @import '../../scss/_mixins';
+  .apos-toolbar {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .apos-toolbar__group--left > * {
+    margin-right: 7.5px;
```

Comment:
these magic numbers skeeve me out

Reply:
using 7.5 in a few places. not magic, just between 5 and 10